Cargo.toml

```diff
@@ -6,15 +6,14 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-anyhow = "1.0.82"
+anyhow = "1.0.82" 
 comat = "0.1.3"
-csv = "1.3.0"
-flate2 = "1.0.30"
-indicatif = "0.17.8"
-nohash = "0.2.0"
+csv = { version = "1.3.0", default-features = false}
+flate2 = { version = "1.0.30"} 
+indicatif = { version = "0.17.8", default-features = false}
+nohash = {version = "0.2.0"} 
```

Comment:
why?

Reply:
nothing in particular, i had disabled default features to see if it works without (the cheap and hacky way to find out xD) and it didn't so i just removed it instead of reverting it to the old form.